### PR TITLE
[DOCS-12567] Updating inconsistent policy names

### PR DESCRIPTION
**Pull request by Heather-MuleSoft**
A bug was discovered in the docs where a policy had two different names on the same page in the Docs and the policy name did not match the name on the release notes page. While determining what the policy name was in the UI to update it correctly, I noticed a couple of other discrepancies in the policy docs that I went ahead and updated as well. To see a full analysis see the Jira issue where I attached a Google doc with a table and some explanations.

### CODE REVIEW

File: modules/ROOT/nav.adoc [hunk 1]
```diff
@@ -70,32 +70,32 @@
   **** xref:disable-edit-remove-task.adoc[Disabling, Editing, or Removing a Policy]
   // *** xref:prepare-raml-task.adoc[Traits & policies concepts of RAML/OAS based APIs]
    *** xref:policies-ootb-landing-page.adoc[Default Policies Directory]
-   **** xref:basic-authentication-simple-concept.adoc[Basic Authentication: Simple]
-   **** xref:basic-authentication-ldap-concept.adoc[Basic Authentication: LDAP]
+   **** xref:basic-authentication-ldap-concept.adoc[Basic Authentication - LDAP]
```
Comment (kbostrom) [inline modules/ROOT/nav.adoc:73]:
```suggestion
   **** xref:basic-authentication-ldap-concept.adoc[Basic LDAP Authentication]
```

File: modules/ROOT/nav.adoc [hunk 2]
```diff
@@ -70,32 +70,32 @@
...
+   **** xref:basic-authentication-simple-concept.adoc[Basic Authentication - Simple]
```
Comment (kbostrom) [inline modules/ROOT/nav.adoc:74]:
```suggestion
   **** xref:basic-authentication-simple-concept.adoc[Basic Authentication]
```

File: modules/ROOT/nav.adoc [hunk 3]
```diff
@@ -70,32 +70,32 @@
...
-   **** xref:policy-ping-federate.adoc[PingFederate OAuth 2.0 Token Enforcement]
+   **** xref:openam-oauth-token-enforcement-policy.adoc[OpenAM Access Token Enforcement]
+   **** xref:policy-openid-connect.adoc[OpenID Connect Access Token Enforcement]
+   **** xref:policy-ping-federate.adoc[PingFederate Access Token Enforcement]
    //**** xref:apply-oauth-token-policy-task.adoc[OAuth 2.0 Token Validation]
    **** xref:rate-limiting.adoc[Rate Limiting]
    //**** xref:configure-rate-limiting-task.adoc[Rate Limiting Policy v1.0.0 or v1.1.0]
    //**** xref:rate-limit-1.2.0-task.adoc[Rate Limiting Policy v1.2.0]
-   **** xref:rate-limiting-sla-policy.adoc[Rate-Limiting SLA]
+   **** xref:rate-limiting-sla-policy.adoc[Rate Limiting - SLA-based]
```
Comment (kbostrom) [inline modules/ROOT/nav.adoc:98]:
```suggestion
   **** xref:rate-limiting-sla-policy.adoc[SLA-Based Rate Limiting]
```

File: modules/ROOT/pages/basic-authentication-ldap-concept.adoc [hunk 4]
```diff
@@ -1,4 +1,4 @@
-= Basic Authentication: LDAP Policy
+= Basic Authentication - LDAP
```
Comment (kbostrom) [inline modules/ROOT/pages/basic-authentication-ldap-concept.adoc:1]:
```suggestion
= Basic LDAP Authentication
```

File: modules/ROOT/pages/basic-authentication-ldap-concept.adoc [hunk 5]
```diff
@@ -13,7 +13,7 @@ endif::[]
 |401 - Unauthorized or invalid client application credentials
 |===
 
-The Lightweight Directory Access Protocol (LDAP) authentication policy specifies how to restrict access to an API using LDAP authentication mechanism. This policy is available only in Mule 4 or later. 
+The Basic Authentication - LDAP authentication policy specifies how to restrict access to an API using the LDAP (Lightweight Directory Access Protocol) authentication mechanism. This policy is available only in Mule 4 or later.
```
Comment (kbostrom) [inline modules/ROOT/pages/basic-authentication-ldap-concept.adoc:16]: If these are actual policy names, perhaps we can change them to be title-style-compliant in the title, but use the actual policy name in the text following?

File: modules/ROOT/pages/basic-authentication-simple-concept.adoc [hunk 6]
```diff
@@ -1,11 +1,11 @@
-= Basic Authentication: Simple Policy
+= Basic Authentication - Simple
```
Comment (kbostrom) [inline modules/ROOT/pages/basic-authentication-simple-concept.adoc:1]: ditto/GLOBAL edit suggestions for topic titles vs. referring to actual policy names in narrative.
I'll stop marking these, now that I've beaten this dead horse!

File: modules/ROOT/pages/client-id-based-policies.adoc [hunk 7]
```diff
@@ -211,9 +213,9 @@ For SOAP applications, the response is a SOAP error.
 
 * xref:manage-client-apps-latest-task.adoc[Managing API instance Contracts on API Manager]
 * xref:access-client-app-id-task.adoc[Getting Client ID and Client Secret of a registered Client App]
-* xref:rate-limiting-sla-policy.adoc[Reviewing Rate-Limiting SLA Policy]
-* xref:policy-ping-federate.adoc[PingFederate OAuth 2.0 Token Enforcement]
-* xref:policy-openid-connect.adoc[OpenID Connect OAuth 2.0 Token Enforcement]
+* xref:rate-limiting-sla-policy.adoc[Reviewing Rate Limiting - SLA-based Policy]
```
Comment (kbostrom) [inline modules/ROOT/pages/client-id-based-policies.adoc:216]:
```suggestion
* xref:rate-limiting-sla-policy.adoc[Reviewing Rate Limiting for an SLA-Based Policy]
```

File: modules/ROOT/pages/cors-policy.adoc [hunk 8]
```diff
@@ -1,24 +1,24 @@
...
 |===
 
-CORS is a mechanism by which a web application can access resources that are defined in another domain. Browsers implement this standard by default. The CORS policy complies with the https://fetch.spec.whatwg.org/[CORS W3C recommendation] standards.
+Cross-Origin Resource Sharing (CORS) is a mechanism by which a web application can access resources that are defined in another domain. Browsers implement this standard by default. The Cross-Origin Resource Sharing policy complies with the https://fetch.spec.whatwg.org/[CORS W3C recommendation] standards.
 
 == Why CORS
 
-CORS enables you to achieve web page security and web integrity in your environment. To know why you might need to apply the CORS policy to your backend, you must first understand _origins_, _cookies_, and how they can be manipulated to violate web page integrity.
+The Cross-Origin Resource Sharing policy enables you to achieve web page security and web integrity in your environment. To know why you might need to apply the Cross-Origin Resource Sharing policy to your backend, you must first understand _origins_, _cookies_, and how they can be manipulated to violate web page integrity.
```
Comment (kbostrom) [inline modules/ROOT/pages/cors-policy.adoc]:
```suggestion
The Cross-Origin Resource Sharing policy enables you to achieve web page security and web integrity in your environment. To know why you might need to apply the Cross-Origin Resource Sharing policy to your backend, you must first understand _origins_ and _cookies_, and how they can be manipulated to violate web page integrity.
```

File: modules/ROOT/pages/cors-policy.adoc [hunk 9]
```diff
@@ -52,9 +52,9 @@ For example, if your bank implements the CORS server-side protocol on its login
 
 == How CORS Works
 
-When your web pages request data, the browser detects whether the request is from within same origin and determines whether to apply the CORS algorithm. If you query data from a web page that is not in your origin, then the CORS policy is applied. 
+When your web pages request data, the browser detects whether the request is from within same origin and determines whether to apply the CORS algorithm. If you query data from a web page that is not in your origin, then the Cross-Origin Resource Sharing policy is applied.
```
Comment (kbostrom) [inline modules/ROOT/pages/cors-policy.adoc]:
```suggestion
When your web pages request data, the browser determines whether the request is from within your origin and if not, applies the CORS algorithm.
```

File: modules/ROOT/pages/cors-policy.adoc [hunk 10]
```diff
@@ -52,9 +52,9 @@ For example, if your bank implements the CORS server-side protocol on its login
 
 == How CORS Works
 
-When your web pages request data, the browser detects whether the request is from within same origin and determines whether to apply the CORS algorithm. If you query data from a web page that is not in your origin, then the CORS policy is applied. 
+When your web pages request data, the browser detects whether the request is from within same origin and determines whether to apply the CORS algorithm. If you query data from a web page that is not in your origin, then the Cross-Origin Resource Sharing policy is applied. 
 
-The CORS algorithm works on the web server and on the client-side for the web page that requested the information. The client-side algorithm in the CORS policy is implemented by:
+The CORS algorithm works on the web server and on the client-side for the web page that requested the information. The client-side algorithm in the Cross-Origin Resource Sharing policy is implemented by:
```
Comment (kbostrom) [inline modules/ROOT/pages/cors-policy.adoc]:
```suggestion
The CORS algorithm works on the web server and on the client side for the web page that requested the information. The client-side algorithm in the Cross-Origin Resource Sharing policy is implemented by:
```

File: modules/ROOT/pages/cors-policy.adoc [hunk 11]
```diff
@@ -195,17 +195,17 @@ When you apply the policy to your API from the UI, the following parameters are
 
 == FAQs
```
Comment (kbostrom) [inline modules/ROOT/pages/cors-policy.adoc:196]: Per style ref.:

FAQ use in documentation
Going forward, rework legacy FAQs as time allows, and do not create new FAQs.

Anything that you anticipate your audience having questions about belongs in narrative context or within a reference or a troubleshooting topic. Ask your editor for assistance.

For those who are interested in a complete explanation of why FAQs violate the CX goal of crafting purposeful information, the following article is excellent:

https://alistapart.com/article/no-more-faqs-create-purposeful-information-for-a-more-effective-user-experi/

File: modules/ROOT/pages/external-oauth-2.0-token-validation-policy.adoc [hunk 12]
```diff
@@ -144,7 +144,7 @@ No. This policy is designed to work exclusively with the xref:mule-oauth-provide
 
 *What should I do if the Mule OAuth provider validation endpoint is rejecting the requests sent by the policy?*
 
-If the validation endpoint is rejecting requests, verify whether the URL configured in the policy is reachable from the machine where the policy is running and that no authentication is required to successfully complete the request. If your use case requires the validation endpoint to be protected, you can switch to the xref:policy-openid-connect.adoc[Open ID Connect policy].
+If the validation endpoint is rejecting requests, verify whether the URL configured in the policy is reachable from the machine where the policy is running and that no authentication is required to successfully complete the request. If your use case requires the validation endpoint to be protected, you can switch to the xref:policy-openid-connect.adoc[Open ID Connect Access Token Enforcement policy].
```
Comment (kbostrom) [inline modules/ROOT/pages/external-oauth-2.0-token-validation-policy.adoc]:
```suggestion
If the validation endpoint is rejecting requests, verify that the URL configured in the policy is reachable from the machine where the policy is running and that no authentication is required to successfully complete the request. If your use case requires the validation endpoint to be protected, you can switch to the xref:policy-openid-connect.adoc[Open ID Connect Access Token Enforcement policy].
```

File: modules/ROOT/pages/openam-oauth-token-enforcement-policy.adoc [hunk 13]
```diff
@@ -18,7 +18,7 @@ endif::[]
 
 IMPORTANT: This policy is only available in a Federated organization configured to use OpenAM as Client Management solution.
 
-The OpenAM OAuth 2.0 Token Enforcement Policy restricts access to a protected resource, by only allowing HTTP requests if the token provided in such request is a valid one and, optionally, the required OAuth scopes are fulfilled.
+The OpenAM Access Token Enforcement policy restricts access to a protected resource, by only allowing HTTP requests if the token provided in such request is a valid one and, optionally, the required OAuth scopes are fulfilled.
```
Comment (kbostrom) [inline modules/ROOT/pages/openam-oauth-token-enforcement-policy.adoc]:
```suggestion
The OpenAM Access Token Enforcement policy restricts access to a protected resource by allowing HTTP requests only if the token provided in such request is a valid one and, optionally, the required OAuth scopes are fulfilled.
```

File: modules/ROOT/pages/openam-oauth-token-enforcement-policy.adoc [hunk 14]
```diff
@@ -45,14 +45,14 @@ Each type of client supports three of the four OAuth grant types defined by the
 
 To improve performance, once a token is validated, by default, the token is cached by the system until it expires. Sometimes it is desirable to reduce or even disable the caching mechanism, for example to take into account revoked token scenarios.
 
-To control token cached time (ie token cache time before retrying validation against the OAuth Provider), you may specify the following property:
+To control token cached time (for example, token cache time before retrying validation against the OAuth Provider), you may specify the following property:
```
Comment (kbostrom) [inline modules/ROOT/pages/openam-oauth-token-enforcement-policy.adoc]:
```suggestion
To control token cached time (for example, token cache time before retrying validation against the OAuth Provider), specify the following property:
```

File: modules/ROOT/pages/policy-openid-connect.adoc [hunk 15]
```diff
@@ -111,7 +111,7 @@ You can obtain the client ID of the OAuth2 token by using the following DataWeav
 
 == Communicating with the Token Validation Endpoint Using a Proxy
 
-You can enable the OpenID Connect OAuth 2.0 Token Enforcement policy to use the gateway proxy settings by specify the following property when starting the Mule runtime engine: 
+You can enable the OpenID Connect Access Token Enforcement policy to use the gateway proxy settings by specify the following property when starting the Mule runtime engine:
```
Comment (kbostrom) [inline modules/ROOT/pages/policy-openid-connect.adoc]:
```suggestion
You can enable the OpenID Connect Access Token Enforcement policy to use the gateway proxy settings by specifying the following property when starting Mule runtime engine: 
```
I know it is wacky to omit the article, but despite it's capitalization, "Mule runtime engine" is the complete product name...sigh....

File: modules/ROOT/pages/policy-ping-federate.adoc [hunk 16]
```diff
@@ -104,7 +104,7 @@ You can obtain the client ID of the OAuth2 token by using the following DataWeav
 
 == Communicating with the Token Validation Endpoint Using a Proxy
 
-You can enable the PingFederate OAuth 2.0 Token Enforcement policy to use the gateway proxy settings by specifying the following property when starting the Mule runtime engine: 
+You can enable the PingFederate Access Token Enforcement policy to use the gateway proxy settings by specifying the following property when starting the Mule runtime engine:
```
Comment (kbostrom) [inline modules/ROOT/pages/policy-ping-federate.adoc]:
```suggestion
You can enable the PingFederate Access Token Enforcement policy to use the gateway proxy settings by specifying the following property when starting Mule runtime engine: 
```

File: modules/ROOT/pages/rate-limiting-sla-policy.adoc [hunk 17]
```diff
@@ -82,11 +82,11 @@ When you configure your Rate-Limiting SLA policy, you must consider certain aspe
 
 === Choosing Between a Cluster or a Standalone Configuration
 
-If you use only one Client application as an authorization mechanism in your server, the same recommendations as in the Rate-Limiting policy applies to your use case. You might have several Mule instances and you might map different client applications to different servers in the load balancer.
+If you use only one Client application as an authorization mechanism in your server, the same recommendations as in the Rate Limiting policy applies to your use case. You might have several Mule instances and you might map different client applications to different servers in the load balancer.
```
Comment (kbostrom) [inline modules/ROOT/pages/rate-limiting-sla-policy.adoc]:
```suggestion
If you use only one client application as an authorization mechanism in your server, the same recommendations as in the Rate Limiting policy apply to your use case. You might have several Mule instances and you might map different client applications to different servers in the load balancer.
```

File: modules/ROOT/pages/rate-limiting-sla-policy.adoc [hunk 18]
```diff
@@ -95,11 +95,11 @@ In a cluster configuration, the nodes must share information across the cluster
...
 
-Therefore, you must set up window sizes greater than one minute in Rate Limiting and Rate-Limiting SLA policy configurations only for a cluster scenario.
+Therefore, you must set up window sizes greater than one minute in Rate Limiting and Rate Limiting - SLA-based policy configurations only for a cluster scenario.
 
-=== Choosing Persistence for Your Rate-Limiting SLA Policy
+=== Choosing Persistence for Your Rate Limiting - SLA-based Policy
 
-You can configure your Rate-Limiting SLA policy to use windows that persist as long as days, months, and years. For example, suppose you want to allow your user to consume 1 million requests per year, but you cannot ensure that the node will be up the entire period or will need maintenance, which may result in restarting the Mule runtime engine.
+You can configure your Rate Limiting - SLA-based policy to use windows that persist as long as days, months, and years. For example, suppose you want to allow your user to consume 1 million requests per year, but you cannot ensure that the node will be up the entire period or will need maintenance, which may result in restarting the Mule runtime engine.
```
Comment (kbostrom) [inline modules/ROOT/pages/rate-limiting-sla-policy.adoc]:
```suggestion
You can configure your Rate Limiting - SLA-based policy to use windows that persist as long as days, months, and years. For example, suppose you want to enable your user to consume 1 million requests per year, but you cannot ensure that the node will be up the entire period or will not need maintenance, which might result in restarting Mule.
```

File: modules/ROOT/pages/rate-limiting-sla-policy.adoc [hunk 19]
```diff
@@ -140,7 +140,7 @@ checked or unchecked
 
 === Configuring Policy Parameters for Anypoint Service Mesh (Non-Mule Applications)
 
-The Rate-Limiting SLA policy works in the same way for Anypoint Service Mesh (non-Mule applications), excluding the following differences:
+The Rate Limiting - SLA-based policy works in the same way for Anypoint Service Mesh (non-Mule applications), excluding the following differences:
```
Comment (kbostrom) [inline modules/ROOT/pages/rate-limiting-sla-policy.adoc]:
```suggestion
The Rate Limiting - SLA-based policy works in the same way for Anypoint Service Mesh (non-Mule applications), but with the following differences:
```

File: modules/ROOT/pages/spike-control-reference.adoc [hunk 20]
```diff
@@ -80,7 +80,7 @@ When you apply the policy to your API from the UI, the following parameters are
 | Method & Resource conditions | The option to add configurations to only a select few or all methods and resources of the API | Yes
 |===
 
-To enforce request quotas, see the xref:rate-limiting-sla-policy.adoc[Rate-Limiting SLA policy] and xref:rate-limiting.adoc[Rate Limiting policy].
+To enforce request quotas, see the xref:rate-limiting-sla-policy.adoc[Rate Limiting - SLA-based policy] and xref:rate-limiting.adoc[Rate Limiting policy].
 
 == FAQ
```
Comment (kbostrom) [inline modules/ROOT/pages/spike-control-reference.adoc:85]: ditto prev. comment about FAQs

**Review by kbostrom — DISMISSED** [review]
CE

File: modules/ROOT/nav.adoc (hunk 1 above)
Comment (Heather-MuleSoft) [reply modules/ROOT/nav.adoc:73]: After talking with Priya and the engineering manager for policies, I think it best to stay consistent with the name that is in the UI.

File: modules/ROOT/pages/policy-ping-federate.adoc (hunk 16 above)
Comment (Heather-MuleSoft) [reply modules/ROOT/pages/policy-ping-federate.adoc]: will fix

File: modules/ROOT/pages/rate-limiting-sla-policy.adoc (hunk 17 above)
Comment (Heather-MuleSoft) [reply modules/ROOT/pages/rate-limiting-sla-policy.adoc]: will fix

File: modules/ROOT/pages/rate-limiting-sla-policy.adoc (hunk 19 above)
Comment (Heather-MuleSoft) [reply modules/ROOT/pages/rate-limiting-sla-policy.adoc]: will fix

File: modules/ROOT/pages/rate-limiting-sla-policy.adoc (hunk 18 above)
Comment (Heather-MuleSoft) [reply modules/ROOT/pages/rate-limiting-sla-policy.adoc]: will update

File: modules/ROOT/pages/cors-policy.adoc (hunk 8 above)
Comment (Heather-MuleSoft) [reply modules/ROOT/pages/cors-policy.adoc]: will fix

File: modules/ROOT/pages/cors-policy.adoc (hunk 9 above)
Comment (Heather-MuleSoft) [reply modules/ROOT/pages/cors-policy.adoc]: will fix

File: modules/ROOT/pages/cors-policy.adoc (hunk 10 above)
Comment (Heather-MuleSoft) [reply modules/ROOT/pages/cors-policy.adoc]: will fix

File: modules/ROOT/pages/cors-policy.adoc (hunk 11 above)
Comment (Heather-MuleSoft) [reply modules/ROOT/pages/cors-policy.adoc:196]: ok, I will open a Gus ticket to weave the FAQs into the rest of the content in the future - thanks Kim

File: modules/ROOT/pages/external-oauth-2.0-token-validation-policy.adoc (hunk 12 above)
Comment (Heather-MuleSoft) [reply modules/ROOT/pages/external-oauth-2.0-token-validation-policy.adoc]: will fix

File: modules/ROOT/pages/openam-oauth-token-enforcement-policy.adoc (hunk 13 above)
Comment (Heather-MuleSoft) [reply modules/ROOT/pages/openam-oauth-token-enforcement-policy.adoc]: will fix

File: modules/ROOT/pages/openam-oauth-token-enforcement-policy.adoc (hunk 14 above)
Comment (Heather-MuleSoft) [reply modules/ROOT/pages/openam-oauth-token-enforcement-policy.adoc]: will fix

File: modules/ROOT/pages/policy-openid-connect.adoc (hunk 15 above)
Comment (Heather-MuleSoft) [reply modules/ROOT/pages/policy-openid-connect.adoc]: will fix

**Review by fermujica — DISMISSED** [review]

**Comment by sanderson-sfdc** [comment]
!gate waiver add terminology

The terminology is getting tripped up by content end-users will not see that is required to ensure proper redirects for customers from old pages to new ones that resolve terminology issues.

The AsciiDoc code looks like this:

modules/ROOT/pages/legacy-ip-allowlist.adoc
5::page-aliases: ip-whitelist.adoc

And it results in a redirects file for nginx that is never exposed to customers.

**Comment by sanderson-sfdc** [comment]
!gate waiver add terminology

**Review by priyaayyar — APPROVED** [review]

**Comment by sanderson-sfdc** [comment]
!gate waiver add terminology

**Comment by mlockhart** [comment]
!gate waiver menu

**Comment by sanderson-sfdc** [comment]
!gate waiver add Terminology

**Pull request merged**